### ISSUE TITLE
Some questions about ResNet.

### ISSUE THREAD

**Issue by vaynelau** [mileyan/simple_shot#3]
It is a really nice job! We tried to use resnet as a backbone just like what you did in our own experiment. However, it didn't improve performance as we expected and even performed worse than 4-conv. I wonder if there are some tricks when you trained your model with a resnet. Thank you very much.

**Comment by mileyan** [comment]
Thanks. We actually did not use any trick during training. Could you check whether you use the hyper-parameters as us?
num-classes=64
batch-size=256
split-dir=./split/mini/
lr 0.1
lr-gamma 0.1
optimizer=SGD
meta-val-interval=4
workers=40
meta-val-metric=cosine
arch=resnet10
save-path=./results/mini/softmax/resnet10
scheduler=multi_step
epochs=90
